app/src/rooms/models.py

```diff
@@ -193,6 +194,12 @@ def unlock(self, sender, leaving=False):
             self.save()
             lock.delete()
 
+    @staticmethod
+    def name_to_key_orderable(room):
+        room_name = room.name.lower()
+        groups = re.split(r"(\d+)", room_name)
+        return tuple(int(g) if re.match(r"\d+", g) else g for g in groups)
```

Comment:
Duplicated method in `app/src/rooms/views.py` in function `list_csv_members_by_room` can be removed and this method used instead :)